contracts/erc721/UserMintableERC721.sol

```diff
@@ -36,6 +42,10 @@ contract UserMintableERC721 is Initializable, ERC721, ERC721Enumerable, ERC721Me
   // Mapping from token details to token ID
   mapping(uint256 => OwnedAnchor) internal _tokenDetails;
 
+  // Value of the Signature purpose for an identity
+  // solium-disable-next-line
+  uint256 constant internal SIGNATURE_PURPOSE = 0x774a43710604e3ce8db630136980a6ba5a65b5e6686ee51009ed5f3fded6ea7e;
```

Comment:
I think we called it signing key in some other places. We should agree on a name. :)

Reply:
This is the purpose not the key

Reply:
I know. 

I just thought of renaming it to.
```suggestion
  uint256 constant internal SIGNING_PURPOSE = 0x774a43710604e3ce8db630136980a6ba5a65b5e6686ee51009ed5f3fded6ea7e;
```

or renaming the other stuff :)